src/store/modules/i18n.js

```diff
@@ -18,6 +18,13 @@ async function loadPages (locale) {
   )
 }
 
+async function loadApi (locale) {

```

Comment:
should this go into a helpers file?

Reply:
I'm not an expert on dynamic imports but I think maybe we can't generalize it or webpack will have trouble statically analyzing which files can be imported.

Reply:
Not to generalize it but to group it with other i18n functions (if applicable).